codegen/src/main/java/software/amazon/awssdk/codegen/CodeGenerator.java

```diff
@@ -32,15 +32,17 @@
 import software.amazon.awssdk.codegen.validation.ModelValidationContext;
 import software.amazon.awssdk.codegen.validation.ModelValidationReport;
 import software.amazon.awssdk.codegen.validation.ModelValidator;
+import software.amazon.awssdk.codegen.validation.SharedModelsValidator;
 import software.amazon.awssdk.codegen.validation.ValidationEntry;
 import software.amazon.awssdk.utils.Logger;
 
 public class CodeGenerator {
     private static final Logger log = Logger.loggerFor(CodeGenerator.class);
     private static final String MODEL_DIR_NAME = "models";
 
-    // TODO: add validators
-    private static final List<ModelValidator> DEFAULT_MODEL_VALIDATORS = Collections.emptyList();
+    private static final List<ModelValidator> DEFAULT_MODEL_VALIDATORS = Collections.singletonList(
```

Comment:
Just to double check, if some validators are manually specified in the `CodeGenerator.Builder` it will override the `DEFAULT_MODEL_VALIDATORS`. Is that what we want?

Reply:
Yes, that's right